src/mcts/params.rs

```diff
@@ -115,4 +144,19 @@ make_mcts_params! {
     material_offset: i32 = 700, 400, 1200, 40, 0.002;
     material_div1: i32 = 32, 16, 64, 3, 0.002;
     material_div2: i32 = 1024, 512, 1536, 64, 0.002;
+    tm_opt_value1: f64 = 0.48, 0.1, 1.0, 0.05, 0.002;
+    tm_opt_value2: f64 = 0.32, 0.1, 1.0, 0.03, 0.002;
+    tm_opt_value3: f64 = 0.60, 0.1, 1.0, 0.06, 0.002;
+    tm_optscale_value1: f64 = 1.25, 0.1, 2.0, 0.1, 0.002;
+    tm_optscale_value2: f64 = 2.5, 0.1, 5.0, 0.15, 0.002;
+    tm_optscale_value3: f64 = 0.50, 0.1, 1.0, 0.05, 0.002;
+    tm_optscale_value4: f64 = 0.25, 0.1, 1.0, 0.02, 0.002;
+    tm_max_value1: f64 = 3.39, 1.0, 10.0, 0.4, 0.002;
+    tm_max_value2: f64 = 3.01, 1.0, 10.0, 0.4, 0.002;
+    tm_max_value3: f64 = 2.93, 1.0, 10.0, 0.4, 0.002;
+    tm_maxscale_value1: f64 = 12.0, 1.0, 24.0, 1.2, 0.002;
+    tm_maxscale_value2: f64 = 4.00, 1.0, 12.0, 0.6, 0.002;
+    tm_bonus_ply: f64 = 11.0, 1.0, 30.0, 1.5, 0.002;
+    tm_bonus_value1: f64 = 0.5, 0.1, 2.0, 0.05, 0.002;
+    tm_max_time: f64 = 0.850, 0.400, 0.990, 0.085, 0.002;
```

Comment:
I'm not sure about having these TM params always exposed as tunable, am open to being convinced otherwise.

Reply:
Since I'm planning to do a timeman tune very soon, I think the params can be removed after the tune instead. currently the params aren't tuned at all and so there should almost certainly be room for gains.

Reply:
@XInTheDark cool, I'm happy with that.